src/node.ts

```diff
@@ -7,5 +7,7 @@ export enum NodeType {
   HttpServer = 'http_server',
   HttpOperation = 'http_operation',
   Model = 'model',
+  Oas2Model = 'oas2_model',
+  Oas3Model = 'oas3_model',
```

Comment:
This node type should match up with what we're storing in the DB not what graphite is using. Agreed it's confusing and we should probably update graphite: https://github.com/stoplightio/graphite/blob/28a15d3ba3b381eb33eb1afdc61c0c55ac88f5c5/src/plugins/oas2/types.ts#L29